packages/design-system/src/components/contextMenu/components/button.js

```diff
@@ -26,6 +26,7 @@ import { useMemo, forwardRef } from '@googleforcreators/react';
  */
 import { Button as BaseButton } from '../../button';
 import { useContextMenu } from '../contextMenuProvider';
+import { Tooltip, TOOLTIP_PLACEMENT } from '../../tooltip';
```

Comment:
See the "Relevant Technical Choices" for the reason of the changes here.
@BrittanyIRL / @samwhale Let me know if you have any concerns with the change.